bentoml/service/__init__.py

```diff
@@ -686,7 +686,7 @@ def pack(self, name, *args, **kwargs):
         :param kwargs: kwargs passing to the target model artifact to be packed
         :return: this BentoService instance
         """
-        self.artifacts.get(name).pack(*args, **kwargs)
+        self.artifacts.get(name).pack(metadata=metadata, *args, **kwargs)
```

Comment:
I am curious about additional `metadata` arguments. `**kwargs` should able to do the job, right?

Reply:
not sure, will double check this later when I have time

Reply:
you're right, this is unneeded! will remove